recipes/theora/all/test_v1_package/test_package.c

```diff
@@ -0,0 +1,138 @@
+/********************************************************************
+ *                                                                  *
+ * THIS FILE IS PART OF THE OggTheora SOFTWARE CODEC SOURCE CODE.   *
+ * USE, DISTRIBUTION AND REPRODUCTION OF THIS LIBRARY SOURCE IS     *
+ * GOVERNED BY A BSD-STYLE SOURCE LICENSE INCLUDED WITH THIS SOURCE *
+ * IN 'COPYING'. PLEASE READ THESE TERMS BEFORE DISTRIBUTING.       *
+ *                                                                  *
+ * THE Theora SOURCE CODE IS COPYRIGHT (C) 2002-2010                *
+ * by the Xiph.Org Foundation and contributors http://www.xiph.org/ *
+ *                                                                  *
+ ********************************************************************
+  function: example of querying various library parameters.
+  last mod: $Id$
+ ********************************************************************/
+
+#ifdef HAVE_CONFIG_H
```

Comment:
This file can be deleted to avoid duplicating it. The CMakeLists.txt can compile the file from `../test_package/test_package.c...`.

Reply:
ok